File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.IO;
using DevExpress.XtraEditors.Repository;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        string HtmlString = @"<html>
<body>

<h1>My First Heading</h1>

<p>My first paragraph.</p>

</body>
</html>
";

        public Form1()
        {
            InitializeComponent();
            myButtonEdit1.EditValue = HtmlString;
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms

Namespace WindowsApplication1

    Public Partial Class Form1
        Inherits Form

        Private HtmlString As String = "<html>
<body>

<h1>My First Heading</h1>

<p>My first paragraph.</p>

</body>
</html>
"

        Public Sub New()
            InitializeComponent()
            myButtonEdit1.EditValue = HtmlString
        End Sub
    End Class
End Namespace
